### PR TITLE
Allows you to edit DTA code on partner banks

### PR DESCRIPTION
**Pull request by open-net-sarl**

### CODE REVIEW

File: l10n_ch_dta/bank_view.xml [hunk 1]
```diff
@@ -6,11 +6,9 @@
       <field name="model">res.partner.bank</field>
       <field name="inherit_id" ref="base.view_partner_bank_form"/>
       <field name="arch" type="xml">
-        <group name="bank" position="after">
-          <group string="DTA options" attrs="{'invisible': [('company_id', '=', False)]}">
```
Comment (yvaucher) [inline l10n_ch_dta/bank_view.xml:10]: company_id = False meant that we wanted to display the config only for the bank of the company and not on the bank of the suppliers or customers.

Is there no way to reproduce this ?

File: l10n_ch_dta/__openerp__.py [hunk 2]
```diff
@@ -30,9 +30,8 @@
              'account_payment_order',
              'document'],
  'data': ['data/account_payment_method.xml',
-          ],
- 'demo': ['../account/test/account_minimal_test.xml',
```
Comment (yvaucher) [inline l10n_ch_dta/__openerp__.py:34]: This was fixed in https://github.com/OCA/l10n-switzerland/pull/299/commits/cd2258ad176bd6554dc3ad1db677d9ae6a8649b2

**Comment by yvaucher** [comment]
You need to rebase this PR.

Could you also explain why we need to be able to edit the `dta_code`

**Comment by open-net-sarl** [comment]
This PR is too old (useless). I close it.

**Pull request closed without merging**